### ISSUE TITLE
训练结果为nan

### ISSUE THREAD

**Issue by Yitianw** [ppogg/YOLOv5-Lite#236]
训练的时候，在Model Summary: 375 layers, 5490092 parameters, 5490092 gradients, 15.8 GFLOPS 这里会卡半个小时，然后才会往下走。并且所有的epoch上box       obj       cls     total 这些都是nan。想请教这是什么原因呢？
数据集应该是没有问题的，在原始yolov5上可以正常运行并且效果不错。

`D:\anaconda\anaconda-setup\envs\v5_lite\lib\site-packages\tqdm\auto.py:21: TqdmWarning: IProgress not found. Please update jupyter and ipywidgets. See https://ipywidgets.readthedocs.io/en/stable/user_install.html
  from .autonotebook import tqdm as notebook_tqdm
github: skipping check (not a git repository)
D:\anaconda\anaconda-setup\envs\v5_lite\lib\site-packages\torch\cuda\__init__.py:104: UserWarning: 
NVIDIA GeForce RTX 3060 with CUDA capability sm_86 is not compatible with the current PyTorch installation.
The current PyTorch install supports CUDA capabilities sm_37 sm_50 sm_60 sm_61 sm_70 sm_75 compute_37.
If you want to use the NVIDIA GeForce RTX 3060 GPU with PyTorch, please check the instructions at https://pytorch.org/get-started/locally/

  warnings.warn(incompatible_device_warn.format(device_name, capability, " ".join(arch_list), device_name))
YOLOv5  2023-5-19 torch 1.8.0 CUDA:0 (NVIDIA GeForce RTX 3060, 12287.5MB)

Namespace(adam=False, artifact_alias='latest', batch_size=32, bbox_interval=-1, bucket='', cache_images=False, cfg='models/v5Lite-g.yaml', data='data/coco128.yaml', device='0', entity=None, epochs=10, evolve=False, exist_ok=False, global_rank=-1, hyp='data/hyp.scratch.yaml', image_weights=False, img_size=[640, 640], label_smoothing=0.0, linear_lr=False, local_rank=-1, multi_scale=False, name='exp', noautoanchor=False, nosave=False, notest=False, project='runs/train', quad=False, rect=False, resume=False, save_dir='runs\\train\\exp39', save_period=-1, single_cls=False, sync_bn=False, total_batch_size=32, upload_dataset=False, weights='weights/v5lite-g.pt', workers=8, world_size=1)
tensorboard: Start with 'tensorboard --logdir runs/train', view at http://localhost:6006/

2023-05-23 11:21:05.441188: I tensorflow/stream_executor/platform/default/dso_loader.cc:49] Successfully opened dynamic library cudart64_110.dll
hyperparameters: lr0=0.001, lrf=0.2, momentum=0.937, weight_decay=0.0005, warmup_epochs=3.0, warmup_momentum=0.8, warmup_bias_lr=0.1, box=0.05, cls=0.5, cls_pw=1.0, obj=1.0, obj_pw=1.0, iou_t=0.2, anchor_t=4.0, fl_gamma=0.0, hsv_h=0.015, hsv_s=0.7, hsv_v=0.4, degrees=0.0, translate=0.1, scale=0.5, shear=0.0, perspective=0.0, flipud=0.0, fliplr=0.5, mosaic=0.2, mixup=0.0
wandb: Install Weights & Biases for YOLOv5 logging with 'pip install wandb' (recommended)

                 from  n    params  module                                  arguments                     
  0                -1  1      3520  models.common.Focus                     [3, 32, 3]                    
  1                -1  1     20736  models.common.RepVGGBlock               [32, 64, 3, 2]                
  2                -1  1     18816  models.common.C3                        [64, 64, 1]                   
  3                -1  1     82432  models.common.RepVGGBlock               [64, 128, 3, 2]               
  4                -1  1    156928  models.common.C3                        [128, 128, 3]                 
  5                -1  1    328704  models.common.RepVGGBlock               [128, 256, 3, 2]              
  6                -1  1    625152  models.common.C3                        [256, 256, 3]                 
  7                -1  1   1312768  models.common.RepVGGBlock               [256, 512, 3, 2]              
  8                -1  1    656896  models.common.SPP                       [512, 512, [5, 9, 13]]        
  9                -1  1   1182720  models.common.C3                        [512, 512, 1, False]          
 10                -1  1     65792  models.common.Conv                      [512, 128, 1, 1]              
 11                -1  1         0  torch.nn.modules.upsampling.Upsample    [None, 2, 'nearest']          
 12           [-1, 6]  1         0  models.common.Concat                    [1]                           
 13                -1  1    189696  models.common.C3                        [384, 128, 3, False]          
 14                -1  1     16640  models.common.Conv                      [128, 128, 1, 1]              
 15                -1  1         0  torch.nn.modules.upsampling.Upsample    [None, 2, 'nearest']          
 16           [-1, 4]  1         0  models.common.Concat                    [1]                           
 17                -1  1    173312  models.common.C3                        [256, 128, 3, False]          
 18                -1  1    147712  models.common.Conv                      [128, 128, 3, 2]              
 19          [-1, 14]  1         0  models.common.Concat                    [1]                           
 20                -1  1    173312  models.common.C3                        [256, 128, 3, False]          
 21                -1  1    147712  models.common.Conv                      [128, 128, 3, 2]              
 22          [-1, 10]  1         0  models.common.Concat                    [1]                           
 23                -1  1    173312  models.common.C3                        [256, 128, 3, False]          
 24      [17, 20, 23]  1     13932  models.yolo.Detect                      [7, [[10, 13, 16, 30, 33, 23], [30, 61, 62, 45, 59, 119], [116, 90, 156, 198, 373, 326]], [128, 128, 128]]
Model Summary: 375 layers, 5490092 parameters, 5490092 gradients, 15.8 GFLOPS

Transferred 474/482 items from weights/v5lite-g.pt
Scaled weight_decay = 0.0005
Optimizer groups: 82 .bias, 82 conv.weight, 79 other
train: Scanning '..\coco128\train' images and labels... 3313 found, 0 missing, 0 empty, 0 corrupted: 100%|██████████| 3313/3313 [00:01<00:00, 2562.24it/s]
train: New cache created: ..\coco128\train.cache
val: Scanning '..\coco128\val' images and labels... 853 found, 0 missing, 0 empty, 0 corrupted: 100%|██████████| 853/853 [00:00<00:00, 2501.45it/s]
val: New cache created: ..\coco128\val.cache
Plotting labels... 

autoanchor: Analyzing anchors... anchors/target = 4.07, Best Possible Recall (BPR) = 0.9999
Image sizes 640 train, 640 test
Using 8 dataloader workers
Logging results to runs\train\exp39
Starting training for 10 epochs...

     Epoch   gpu_mem       box       obj       cls     total    labels  img_size
       0/9     2.44G       nan       nan       nan       nan        45       640: 100%|██████████| 104/104 [16:10<00:00,  9.33s/it]  
D:\anaconda\anaconda-setup\envs\v5_lite\lib\site-packages\torch\optim\lr_scheduler.py:129: UserWarning: Detected call of `lr_scheduler.step()` before `optimizer.step()`. In PyTorch 1.1.0 and later, you should call them in the opposite order: `optimizer.step()` before `lr_scheduler.step()`.  Failure to do this will result in PyTorch skipping the first value of the learning rate schedule. See more details at https://pytorch.org/docs/stable/optim.html#how-to-adjust-learning-rate
  warnings.warn("Detected call of `lr_scheduler.step()` before `optimizer.step()`. "
               Class      Images      Labels           P           R      mAP@.5  mAP@.5:.95: 100%|██████████| 14/14 [00:18<00:00,  1.32s/it]
                 all         853        1651     0.00068      0.0013    9.67e-06    1.15e-06

     Epoch   gpu_mem       box       obj       cls     total    labels  img_size
       1/9     6.23G       nan       nan       nan       nan        60       640: 100%|██████████| 104/104 [00:41<00:00,  2.49it/s]
               Class      Images      Labels           P           R      mAP@.5  mAP@.5:.95: 100%|██████████| 14/14 [00:05<00:00,  2.64it/s]
                 all         853        1651     0.00069      0.0311    7.82e-05     1.4e-05

     Epoch   gpu_mem       box       obj       cls     total    labels  img_size
       2/9     6.23G       nan       nan       nan       nan        44       640: 100%|██████████| 104/104 [00:41<00:00,  2.48it/s]
               Class      Images      Labels           P           R      mAP@.5  mAP@.5:.95: 100%|██████████| 14/14 [00:05<00:00,  2.51it/s]
                 all         853        1651    0.000497      0.0247     5.8e-05    8.77e-06

     Epoch   gpu_mem       box       obj       cls     total    labels  img_size
       3/9     6.23G       nan       nan       nan       nan        41       640: 100%|██████████| 104/104 [00:41<00:00,  2.49it/s]
               Class      Images      Labels           P           R      mAP@.5  mAP@.5:.95: 100%|██████████| 14/14 [00:03<00:00,  4.56it/s]
                 all         853        1651    0.000582     0.00486    2.79e-05    5.07e-06

     Epoch   gpu_mem       box       obj       cls     total    labels  img_size
       4/9     6.23G       nan       nan       nan       nan        45       640: 100%|██████████| 104/104 [00:41<00:00,  2.49it/s]
               Class      Images      Labels           P           R      mAP@.5  mAP@.5:.95: 100%|██████████| 14/14 [00:04<00:00,  2.93it/s]
                 all         853        1651    2.66e-06    0.000603    3.53e-08    7.06e-09

     Epoch   gpu_mem       box       obj       cls     total    labels  img_size
       5/9     6.23G       nan       nan       nan       nan        36       640: 100%|██████████| 104/104 [00:41<00:00,  2.53it/s]
               Class      Images      Labels           P           R      mAP@.5  mAP@.5:.95: 100%|██████████| 14/14 [00:05<00:00,  2.43it/s]
                 all         853        1651    0.000489      0.0206     9.1e-05    1.25e-05

     Epoch   gpu_mem       box       obj       cls     total    labels  img_size
       6/9     6.23G       nan       nan       nan       nan        47       640: 100%|██████████| 104/104 [00:41<00:00,  2.53it/s]
               Class      Images      Labels           P           R      mAP@.5  mAP@.5:.95: 100%|██████████| 14/14 [00:02<00:00,  4.80it/s]
                 all         853        1651    0.000583      0.0151    3.62e-05    7.97e-06

     Epoch   gpu_mem       box       obj       cls     total    labels  img_size
       7/9     6.23G       nan       nan       nan       nan        41       640: 100%|██████████| 104/104 [00:41<00:00,  2.54it/s]
               Class      Images      Labels           P           R      mAP@.5  mAP@.5:.95: 100%|██████████| 14/14 [00:05<00:00,  2.44it/s]
                 all         853        1651    0.000564      0.0151    3.38e-05    7.78e-06

     Epoch   gpu_mem       box       obj       cls     total    labels  img_size
       8/9     6.23G       nan       nan       nan       nan        36       640: 100%|██████████| 104/104 [00:41<00:00,  2.53it/s]
               Class      Images      Labels           P           R      mAP@.5  mAP@.5:.95: 100%|██████████| 14/14 [00:05<00:00,  2.53it/s]
                 all         853        1651    0.000406      0.0164    4.11e-05    6.18e-06`
